cmd/restore.go

```diff
@@ -54,24 +64,19 @@ type restoreCommand struct {
 	restoreLog           string
 	includeStatusHistory bool
 
-	connect    func(info db.DialInfo) (core.Database, error)
-	openBackup func(path, tempRoot string) (core.BackupFile, error)
-
 	// manualAgentControl determines if 'juju-restore' or the operator
 	// manages - stops and starts juju and mongo agents - on
 	// other, non-primary controller nodes.
 	// If true, the control is manual and 'juju-restore' will do nothing
 	// to other controller nodes.
 	manualAgentControl bool
 
-	ui          *UserInteractions
-	restorer    *core.Restorer
-	converter   func(member core.ReplicaSetMember) core.ControllerNode
-	readOneChar func(*cmd.Context) (string, error)
+	ui       *UserInteractions
+	restorer *core.Restorer
 
-	// To be used as an option during development to enable an easier way to re-start
-	// all agents in HA federation.
-	// XXXXXXXXXXXXX Remove ME
+	// To be used as an option during development to enable an easier
+	// way to re-start all agents in HA federation.  XXXXXXXXXXXXX
+	// Remove ME
 	restart bool
```

Comment:
What about just putting that arg behind a feature flag?

Reply:
Yeah, good idea - I'll do that.